housing_cooperative_base/models/lease.py

```diff
@@ -106,6 +116,22 @@ def _compute_lease_end(self):
             else:
                 lease.end = lease.expected_end
 
+    @api.multi
+    @api.depends("start", "end")
+    def _compute_state(self):
+        today = fields.Date.today()
+        if self.start and self.end:
```

Comment:
This method is decorated by a `multi`, you should therefore loop on self.

Reply:
Well noticed!